guide/portuguese/html/elements/aside-tag/index.md

```diff
@@ -1,11 +1,27 @@
 ---
 title: Aside Tag
-localeTitle: Além de Tag
+localeTitle: Tag 'Aside'
 ---
-## Além de Tag
```

Comment:
A explicação está ótima mas o texto precisa seguir o formato recomendado para os guides. Precisa de um título no início e de uma sessão de "Mais Informações" no final com links.

Reply:
Repus o título original e acrescentei a secção de Mais informações no final para ficar de acordo com o formato recomendado!